hera_mc/cm_hookup.py

```diff
@@ -39,6 +39,7 @@ def __init__(self, session=None, hookup_list_to_cache=['HH']):
         self.part_type_cache = {}
         self.hookup_local_file = os.path.expanduser('~/.hera_mc_hookup_cache.npy')
         self.hookup_list_to_cache = hookup_list_to_cache
+        self.hookup_dict_library = None
```

Comment:
I think it would be better to name this `saved_hookup_dict` or `cached_hookup_dict`.